Building debian package with cmake build system

I've been maintaining a private debian package for work and wanted to switch it to use the cmake build. I get that it's still very early in the cmake support, but it builds much faster (at least with ninja backend) so I'd like to start using it everywhere. 

I added the files to my fork [here](https://github.com/tmayoff/OpenDDS/tree/debian/debian).

 I'm getting some errors,

using this `debuild -us -uc -i` command to build the package
```
'/usr/bin/perl' '/home/tyler/src/OpenDDS/cmake/scrape_gnuace.pl' '--workspace' '/home/tyler/src/OpenDDS/obj-x86_64-linux-gnu/ace_tao' '--loc-base' '/home/tyler/src/OpenDDS/obj-x86_64-linux-gnu' '--ace' '/home/tyler/src/OpenDDS/obj-x86_64-linux-gnu/ace_tao' '--tao' '/home/tyler/src/OpenDDS/obj-x86_64-linux-gnu/ace_tao/TAO'
make[2]:Leaving directory '/home/tyler/src/OpenDDS/obj-x86_64-linux-gnu/ace_tao'
 is missing dir at /home/tyler/src/OpenDDS/cmake/scrape_gnuace.pl line 121.
CMake Error at cmake/build_ace_tao.cmake:39 (execute_process):
  execute_process failed command indexes:

    1: "Child return code: 255"

Call Stack (most recent call first):
  CMakeLists.txt:47 (include)
```

I'm not sure if this a cmake, debuild or OpenDDS issue but I wasn't even able to download ace_tao without this flag `-DFETCHCONTENT_FULLY_DISCONNECTED=OFF`.

Any help would be appreciated, also side note, would it be useful to merge the debian package support upstream?

--- a/cmake/scrape_gnuace.pl
+++ b/cmake/scrape_gnuace.pl
@@ -42,7 +42,7 @@ my $gnumf = 'GNUmakefile';
 
 sub get_projects {
   my %projects;
-  open(my $fh, '-|', "$make -f $gnumf project_name_list") or die("get_projects make: $!");
+  open(my $fh, '-|', "$make --no-print-directory -f $gnumf project_name_list") or die("get_projects make: $!");
   while (my $project = <$fh>) {
     $project =~ s/\s//;
     my $mf = "$gnumf.$project";

